mobify/progressive-web-scaffold
# Sdk 0.0.6

Updates the SDK to v0.0.6 and pulls in some other changes.

## Changes
- progressive-web-sdk -> 0.0.6

## How to test-drive this PR
- `npm run dev`
- Preview merlins potions
- Ensure the scaffold runs as expected
